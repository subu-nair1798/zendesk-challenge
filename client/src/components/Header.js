import React from 'react';
import { Menu } from 'semantic-ui-react';
import { Link } from 'react-router-dom';

const Header = () => {
    return(
        <Menu style={{ marginTop: "15px" }}>
            <Menu.Item>
                <Link to="/">Zendesk Ticket Viewer</Link>
            </Menu.Item>
            <Menu.Menu position="right">
                <Menu.Item>
                    <Link to="/">Home</Link>
                </Menu.Item>
            </Menu.Menu>
        </Menu>
    );
}

export default Header;