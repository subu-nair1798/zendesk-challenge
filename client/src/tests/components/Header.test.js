import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('<Header />', () => {
    it('renders Page Header', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper).toMatchSnapshot();
    });
});