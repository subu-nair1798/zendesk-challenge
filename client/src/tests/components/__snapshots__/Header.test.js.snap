// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> renders Page Header 1`] = `
<Menu
  style={
    Object {
      "marginTop": "15px",
    }
  }
>
  <MenuItem>
    <Link
      to="/"
    >
      Zendesk Ticket Viewer
    </Link>
  </MenuItem>
  <MenuMenu
    position="right"
  >
    <MenuItem>
      <Link
        to="/"
      >
        Home
      </Link>
    </MenuItem>
  </MenuMenu>
</Menu>
`;
